src/System.IO.StreamSourceGeneration/src/System.IO.StreamSourceGeneration.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>latest</LangVersion>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<DefineConstants Condition="'$(LaunchDebugger)' == 'true'">$(DefineConstants);LAUNCH_DEBUGGER</DefineConstants>
+	</PropertyGroup>
```

Comment:
Probably doesn't need to be committed.

Reply:
Same reply as https://github.com/dotnet/runtimelab/pull/2185#discussion_r1107754088.